### PR TITLE
fix: content with new line

### PR DESCRIPTION
**Pull request by ymarsal-kpa**

### CODE REVIEW

File: base-integration/src/utilities/helper.ts [hunk 1]
```diff
@@ -6,6 +6,10 @@ export class Helper {
 
     let finalData = data.trim();
 
+    finalData = finalData.replace('\r\n', ' ')
+    finalData = finalData.replace('\r', ' ')
+    finalData = finalData.replace('\n', ' ')
+
```
Comment (schwamman) [inline base-integration/src/utilities/helper.ts:12]:
```suggestion
    const regex = /\/r|\/n/g
    finalData = finalData.replace(regex, ' ')
```

**Review by schwamman — CHANGES_REQUESTED** [review]
Can you update this to use a regular expression? Or change it to `replaceAll`? The `replace` function will only update the first matching value if it's given a string.

File: base-integration/src/utilities/helper.ts (hunk 1 above)
Comment (trevorhibblen) [reply base-integration/src/utilities/helper.ts:12]: @schwamman we should prob. keep the original format, that way `\r\n` doesn't create two spaces.
```suggestion
    finalData = finalData.replace(/\r\n/g, ' ')
    finalData = finalData.replace(/\r/g, ' ')
    finalData = finalData.replace(/\n/g, ' ')

```

**Review by schwamman — APPROVED** [review]
✅

**Pull request merged**